README.md

```diff
@@ -89,7 +89,7 @@ For examples & further details about the export format, please refer to [this se
 
 ### Prerequisites
 
-Python 3.6 (or higher) and [pip](https://pip.pypa.io/en/stable/) are required to install docTR. Additionally, you will need to install at least one of [TensorFlow](https://www.tensorflow.org/install/) or [PyTorch](https://pytorch.org/get-started/locally/#start-locally).
+Python 3.6 (or higher) and [pip](https://pip.pypa.io/en/stable/) are required to install docTR. Additionally, you will need to install at least one of [TensorFlow](https://www.tensorflow.org/install/) or [PyTorch](https://pytorch.org/get-started/locally/#start-locally). If you use tensorflow, you will need to install the additional package `tensorflow-addons`
```

Comment:
What do you think about always specifying the extra with the pip install command?
Because here we say that TF needs `tensorflow-addons` but say tomorrow there are 5 extra TF-specific deps, we don't want to leave the user alone to do it :/ 

But if they run `pip install python-doctr[tf]` that will automatically install the remaining dependencies. What do you think?

Reply:
I find it a bit awkward that you have "you should install tensorflow first" and "you can then install the latest release with pip install python-doctr" while it does not produce a working installation. I understand that you propose a solution immediately after,  but doesn't it mean that python-doctr setup alone should always need an argument ?

Anyway, that's a very minor issue and the main correction was the error in the example, so I guess we can cancel this PR if you do not like this correction ^^"

Reply:
Yup I would argue it should, and thus we can remove the section about "you need to install a framework first" :+1: 

Oh don't take it the wrong way, I was thinking about trying to harmonize installation instructions. I'm only trying to find the best option to tackle the underlying installation issue here

Reply:
No worries! I was just saying that messing with setup tool is probably a bit more than twiddling the doc, but we can keep it in the PR if you want. If I understood you correctly, setup_tools should throw an error if an argument is not specified ?

Reply:
Actually, thinking a bit more, I guess it makes sense to have a framework agnostic version, even if it's not standalone. We could simply have a warning in the installation instruction. What do you think about the current state of things ?

Reply:
Yup I was only suggesting to modify the instructions in the README here :) 
We'll have to do the same with the instructions in the documentation but that can wait for another PR